src/uct/api/uct.h

```diff
@@ -1213,6 +1214,53 @@ struct uct_md_attr {
 };
 
 
+/**
+ * @ingroup UCT_MD
+ * @brief UCT MD memory attributes field mask
+ *
+ * The enumeration allows specifying which fields in @ref uct_md_mem_attr_t
+ * are present.
+ */
+enum uct_md_mem_attr_field {
+    UCT_MD_MEM_ATTR_MEM_TYPE = UCS_BIT(0),
```

Comment:
UCT_MD_MEM_ATTR_FIELD_TYPE 
pls add doxygen to each bit

Reply:
Done